### PR TITLE
Launch game from ConsoleUI

### LINKED ISSUE #3534: [Feature] Ability to Launch KSP from consoleui
Problem
-------
I'm running linux and the gui on linux honestly looks horrible, so instead im using the consoleui. but one thing I noticed was missing Is the "Launch KSP" button the gui has. SO I have topen steam seperately and launch ksp through steam

Suggestions
-----------
Just add the option to launch KSP from the consoleui. Best way would probably to add it in the menu above game instance settings

### PR DESCRIPTION
**Pull request by PizzaRules668**
I made console UI launch the game with f2, fixes #3534 
I still need to internationalize the incompatible Mod Warning, What would be the best way to do that?
Also hows it looking so far

### CODE REVIEW

**Comment by HebaruSan** [comment]
Hi and thanks for another submission! I will try to get the first round of code review done within the next week or so.

You don't have to worry about internationalization of Core or ConsoleUI code yet because #3482 is not yet merged to master; you only need to worry about being compatible with the code that's already in master. If your PR merges first, I will update that other PR to internationalize any new code.

**Comment by PizzaRules668** [comment]
I didn't even know that was planned, or being worked on

**Comment by HebaruSan** [comment]
> I didn't even know that was planned, or being worked on

Yup! You can see everything we're currently working on here:

- https://github.com/KSP-CKAN/CKAN/pulls

**Comment by PizzaRules668** [comment]
Looking back at some of the code I don't think I'm ready for a code review quite yet

**Comment by PizzaRules668** [comment]
I think its ready
@HebaruSan

**Comment by HebaruSan** [comment]
About 114 tests are failing with null reference errors. They should pass before code review starts.

File: ConsoleUI/ModListScreen.cs [hunk 1]
```diff
@@ -210,10 +216,6 @@ public ModListScreen(GameInstanceManager mgr, bool dbg, ConsoleTheme regTheme)
                 return true;
             });
 
-            moduleList.AddTip("F8", "Mark auto-installed",
-                () => moduleList.Selection != null && !moduleList.Selection.IsDLC
-                    && (!registry.InstalledModule(moduleList.Selection.identifier)?.AutoInstalled ?? false)
-            );
```
Comment (HebaruSan) [inline ConsoleUI/ModListScreen.cs:216]: Please don't remove this tip.

File: Core/GUIConfiguration.cs [hunk 2]
```diff
@@ -3,6 +3,8 @@
 using System.IO;
```
Comment (HebaruSan) [inline Core/GUIConfiguration.cs:3]: `GUIConfiguration.cs` has "GUI" in its name, it should not be moved from GUI to Core.

File: ConsoleUI/ModListScreen.cs (hunk 1 above)
Comment (PizzaRules668) [reply ConsoleUI/ModListScreen.cs:216]: Its duplicate

Comment (HebaruSan) [reply ConsoleUI/ModListScreen.cs:216]: No, it's not a duplicate.

**Comment by HebaruSan** [comment]
Hi @PizzaRules668, you can run `./build test` (Linux) or `.\build.ps1 test` (Windows) to run the tests locally. They need to pass before we can merge.

**Comment by HebaruSan** [comment]
Hi @PizzaRules668, if you have a chance to come back to this and fix the failing tests, please let us know and we can re-open this. Closing as incomplete and inactive.

**Pull request closed without merging**